opencti-platform/opencti-graphql/src/migrations/1704792752287-user-confidence-level.js

```diff
@@ -0,0 +1,71 @@
+import { elUpdateByQueryForMigration, elUpdateIndicesMappings } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { DatabaseError } from '../config/errors';
+
+export const up = async (next) => {
+  await elUpdateIndicesMappings({
+    user_confidence_level: {
+      dynamic: 'strict',
+      properties: {
+        max_confidence: {
+          coerce: false,
+          type: 'integer'
+        },
+        overrides: {
+          dynamic: 'strict',
+          type: 'nested',
+          properties: {
+            max_confidence: {
+              coerce: false,
+              type: 'integer'
+            },
+            entity_type: {
+              type: 'text',
+              fields: {
+                keyword: {
+                  normalizer: 'string_normalizer',
+                  ignore_above: 512,
+                  type: 'keyword'
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+  });
```

Comment:
This mapping is the one generated by the platform based on the schema (I've checked in kibana).